File: src/main/java/ru/arty_bikini/crm/dto/UserDTO.java
package ru.arty_bikini.crm.dto;

import ru.arty_bikini.crm.data.UserGroup;

//для передали данных в интернете
public class UserDTO {
    private int id;
    private String login;
    private UserGroup group;


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getLogin() {
        return login;
    }

    public void setLogin(String login) {
        this.login = login;
    }

    public UserGroup getGroup() {
        return group;
    }

    public void setGroup(UserGroup group) {
        this.group = group;
    }
}

File: common/src/commonMain/kotlin/ru/arty_bikini/crm/dto/UserDTO.kt
package ru.arty_bikini.crm.dto

import kotlinx.serialization.Serializable
import ru.arty_bikini.crm.dto.enums.TypeWork
import ru.arty_bikini.crm.dto.enums.UserGroup

//для передали данных в интернете
@Serializable
class UserDTO : EntityDTO() {
    override var id = 0
    var login: String? = null
    var group: UserGroup? = null
    var specialisation: String? = null
    var productivityComment : String? = null
    var baseProductivity : Int? = 0;//основная продуктивность(для расклейщиц)
    var productivity : Map<TypeWork, Int>? = null
    
}